README.md

```diff
@@ -17,6 +17,20 @@ You can find the assets in the following locations:
 - [Nebari stacked logo mark](./logo-mark/stacked/)
 - [Nebari logo mark with colored backgrounds](./logo-mark/colored-background/)
 
+## Nebari brand
+
+The Nebari brand uses the following primary colors:
+
+- Yellow: #EAB54E
+- Green: #20AAA1
+- Purple: #BA18DA
+- Black: #0F1015
+
+And, the following fonts:
+
+- Atkinson Hyperlegible
```

Comment:
Suggestion: add links to the fonts in Google Fonts

Reply:
That's a good idea, done!